bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -45,7 +45,7 @@ class ActionsBar extends PureComponent {
     } = recordSettingsList;
 
     const actionBarClasses = {};
-    const { enableExternalVideo } = Meteor.settings.public.app;
+    const { enabled: enableExternalVideo } = Meteor.settings.public.externalVideoPlayer;
```

Comment:
```suggestion
    const { enabled: allowExternalVideo } = Meteor.settings.public.externalVideoPlayer;
```

Reply:
so you can simplify a few lines down

`- allowExternalVideo: enableExternalVideo,`
`+ allowExternalVideo,`